code/modules/modular_armor/modular.dm

```diff
@@ -544,20 +558,27 @@
 		return TRUE
 
 	attachment.do_detach(user, src)
-	update_overlays()
+	update_icon()
 	return TRUE
 
 
 /obj/item/clothing/head/modular/update_overlays()
 	. = ..()
+	/*
 	if(installed_module)
-		. += image(installed_module.icon, installed_module.item_state)
+		. += image(installed_module.icon, ITEM_STATE_IF_SET(installed_module))*/ //these look whack because they dont have non-singl direction spritos.
```

Comment:
don't comment out code. Please fix

Reply:
need sprite and dindt work before, its there if someone adds 1dir sprite for them for obj.

Reply:
then ask or make sprites?

Reply:
i mean it was already borked like that but ok

Reply:
for the last time splinter, remove that bloody code so i can merge!!!

And no, i can't suggest because there are deleted lines in the middle :)